Formula/infer.rb

```diff
@@ -30,8 +30,10 @@ def install
       odie "infer: --without-clang and --without-java are mutually exclusive"
     end
 
-    # fix symbol not found issue (_clock_gettime) on el_capitan
-    ENV.delete("SDKROOT")
+    if MacOS.version == :el_capitan
+      # fix symbol not found issue (_clock_gettime) on el_capitan
+      ENV.delete("SDKROOT")
```

Comment:
This can actually just be removed now since we made setting SDKROOT opt-in instead of opt-out, meaning this is a no-op.

Reply:
Done.